instruqt-tracks/nomad-governance/track_scripts/setup-nomad-client-1

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euvo pipefail
+
+while [ ! -f /opt/instruqt/bootstrap/host-bootstrap-completed ]
+do
+    echo "Waiting for Instruqt to finish booting the virtual machine"
+    sleep 1
+done
+
+# Ensure we always have fresh copies of the Consul 
+# Enterprise license, which we store as Instruqt secrets
+# We don't need a Nomad license on the clients, it fetches
+# from the Nomad server once it can join
+
+# This trick with awk keeps the contents of the secret
+# environment variables from being logged
+echo "Writing license files"
+awk 'BEGIN {print ENVIRON["HC_CONSUL_LICENSE"]}' > /var/consul-license.hclic < /dev/null
```

Comment:
Is it worth doing this?  The license itself is necessarily exposed to participants running the track; I don't think we're too worried about Instruqt swiping and using our licenses (given that they run the platform they're contained in anyway).  I can also think of cases where the license being echoed to the logs would assist in quicker troubleshooting of license-related issues.

Reply:
This keeps it out of the logs because this is a public repo and the logs from actions are publicly viewable. If we care about the license contents for debugging, we can do `consul license inspect`; but longer term my plan is to store the licenses in Instruqt secrets, maintained by a private repo, which can do nightly syncing as well as nightly test to notify us when the license has N days left.